src/param.h

```diff
@@ -44,8 +44,8 @@ struct Param {
 
   static constexpr double kNeuriteDefaultTension = 0.0;
 
-  static constexpr double kNeuriteMinLength = 2.0;
-  static constexpr double kNeuriteMaxLength = 15;
+  double kNeuriteMinLength = 2.0;
+  double kNeuriteMaxLength = 15;
```

Comment:
Hi jean, why did you make these two parameters non-static?

Reply:
If you need to modify these parameters, then maybe make them all the neurite variables non-static? If later you or someone else wants to change for example the `kNeuriteDefaultTension`, then it would require another commit. Also keep in mind that the naming convention for non-static data members differs from static ones.